lib/streamlit/util.py

```diff
@@ -126,15 +126,17 @@ def index_(iterable, x) -> int:
     -------
     int
     """
-
+    if x is None:
+        return 0
```

Comment:
Hm, I don't think this is the behavior we want here as now the function can't be used to find the index of `None` in an iterable. It also gives an incorrect result whenever `x is None` and there is no `None` in the iterable. I'd expect it to throw a `ValueError` rather than return 0.

I think in this case, we'll probably want to change the elif case to something like `isinstance(value, float) and isinstance(x, float)` before we try to subtract the two to avoid a `TypeError` in case `x is None`